src/instance.ts

```diff
@@ -16,12 +15,10 @@ export class VMInstance {
   public instance?: WebAssembly.Instance;
   public backend: IBackend;
   public bech32: BechLib;
-  public eddsa: eddsa;
 
   constructor(backend: IBackend) {
     this.backend = backend;
     this.bech32 = bech32;
-    this.eddsa = new eddsa('ed25519');
```

Comment:
`eddsa` is not serialisable, but nor does it (seem to) hold any state - its methods are pure functions.